### DISCUSSION TITLE
Spectrum 2 icon set preview

### DISCUSSION THREAD

**Discussion by joebudi** [adobe/spectrum-css#2549] [General]
Hi. We're huge fans of the spectrum design system at Budibase. We're excited for the new release. We're evaluating our current icon set (Old Spectrum icons) and wanted to know if there was anywhere publicly where we could view the new icon set.

**Comment by dannify** [comment]
Nothing is available yet as Spectrum 2 is still a work in progress. We currently do not have a timeline for the release of icons but information will be available on the website and release notes as we make progress.

**Reply by joebudi** [reply]
Thank you for the update. Keep up the great work. Spectrum 1 was incredible; we're very excited for Spectrum 2.
